### ISSUE TITLE
FNX-4970 ⁃ Browser UI a11y

### ISSUE THREAD

**Issue by boek** [mozilla-mobile/fenix#135]
Parent: #116

**Comment by sblatz** [comment]
Per @colintheshots's recommendation I'll be running [Accessibility Scanner](https://support.google.com/accessibility/android/answer/6376570?hl=en) as well in order to ensure I'm not missing anything with manual testing.
